man/udevadm.xml

```diff
@@ -564,6 +564,18 @@
             <xi:include href="version-info.xml" xpointer="v251"/>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><option>-i</option></term>
+          <term><option>--include-parents</option></term>
+          <listitem>
+            <para>Trigger parent devices of filtered devices even if the parents
```

Comment:
`filtered devices` sounds ambiguous for me. Maybe, better to `enumerated devices` or `found devices`